src/makeRobot.js

```diff
@@ -38,7 +38,73 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value < 0 || typeof value !== 'number') {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0 && typeof value === 'number') {
        this.coords.y += value;
      }

      return this;
    },
```

Let's make your code shorter